### PR TITLE
Improve masthead

### LINKED ISSUE #97: Improve InstructLab masthead
A masthead contains global properties such as logotype, navigation and settings in an organized fashion and it is accessible from all parts of the application. All pages should share a common masthead. The current masthead emphasizes the dark/light mode switcher and logout button. These items are not the primary actions to expose to the user at this level. The masthead should display the following: 
- Show/hide menu
- Logotype
- Help: quick access to documentation, customer support resources, and link to the About modal.  
- User menu: includes an avatar and the username of the currently logged in user. User dropdown includes application settings such as display language, theme, etc., and Log out.

Reference: https://www.patternfly.org/components/masthead/design-guidelines#elements

@beaumorley

### PR DESCRIPTION
**Pull request by aevo98765**
Fixes #97

### CODE REVIEW

**Comment by aevo98765** [comment]
Screenshot 2024-08-19 at 21 24 41
Screenshot 2024-08-19 at 21 25 17
Screenshot 2024-08-19 at 21 25 23

**Comment by aevo98765** [comment]
Code for review. @Misjohns I am not sure if you are running the dev branches to see the updated UI. If not we could arrange for me to call and I could share screen. I have attached some screenshots which I believe address what you were asking for.

Very open as usual to feedback on this one!

File: src/components/UserMenu/UserMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  return (
+    <Dropdown
+      isOpen={isOpen}
+      onSelect={onSelect}
+      onOpenChange={(isOpen: boolean) => setIsOpen(isOpen)}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle ref={toggleRef} aria-label="user menu dropdown" variant="plain" onClick={onToggleClick} isExpanded={isOpen}>
+          <Flex spaceItems={{ default: 'spaceItemsSm' }} style={{ display: 'flex', alignItems: 'center' }}>
+            <FlexItem>
```
Comment (vishnoianil) [inline src/components/UserMenu/UserMenu.tsx:30]: @aevo98765 Overall the PR looks good, but once i login through the github, it doesn't show the github username or the avatar, it shows the one that is hardcoded here. I think you need to pull the session and use the session.user.name and session.user.image. I locally did the following changes and it worked for me. here is the patch file
https://gist.github.com/vishnoianil/240d67469823365435f128c33717520c

**Review by vishnoianil — CHANGES_REQUESTED** [review]
PR looks good, thanks for the awesome work. One comment regarding the github login avatar and username. Once that's done, i think it's good to go.

**Comment by vishnoianil** [comment]
Forget to mention another issue. Clicking on Documentation, opens the document in the same tab. In my opinion, it's good to open the documentation in the new tab.

File: src/components/UserMenu/UserMenu.tsx (hunk 1 above)
Comment (aevo98765) [reply src/components/UserMenu/UserMenu.tsx:30]: Done

**Comment by aevo98765** [comment]
@vishnoianil done.

**Review by nerdalert — APPROVED** [review]

**Review by vishnoianil — APPROVED** [review]

**Comment by Misjohns** [comment]
The InstructLab icon looks a bit tight in the masthead. Not sure what your masthead height is but for a 76px height I was using 16px padding around the logo.
image

**Pull request merged**